plugins/modules/packaging/os/pulp_repo.py

```diff
@@ -117,7 +129,7 @@
     type: str
   repo_type:
     description:
-      - Repo plugin type to use (i.e. C(rpm), C(docker)).
+      - Repo plugin type to use (i.e. C(rpm), C(deb)).
```

Comment:
Is `docker` not a valid value? Or why did you remove it?

Reply:
Hi. It was never a valid value. Only 'rpm' was ever supported. Having it there made reading this confusing as to what was supported